provisioning/roles/cuttlefish-app/tasks/main.yml

```diff
@@ -92,3 +92,11 @@
     owner: root
     group: root
     mode: 0744
+
+- name: Create old email archiving daily cronjob for 1am
```

Comment:
Nice one! Good analysis :+1: 

You might like to double-check that this is going to run at the time you expect. Since we're not the only ones using this server I'm pretty sure it's set to UTC, not AEST. Does this mean the job will run at 01:00 UTC? Is that the time that matches up to your analysis?